buildtest/buildsystem/base.py

```diff
@@ -252,7 +252,7 @@ def __init__(self, name, recipe_config, config_file=None, testdir=None):
             )
 
         # Start with just terminal logger (changes to file on run)
-        self._init_logger(to_file=False)
+        # self._init_logger(to_file=False)
```

Comment:
@shahzebsiddiqui this will break other tests because this first logger is _just_ to the terminal, the one set later is to file. The logger needs to be present at this stage in the build. If you change one logger (to file) you just need to change the other logger (to the terminal) to do the same. Commenting this out will break things.

Reply:
Yea i kinda realized this and wanted to get rid of the log output from stream shown in my first message in PR. I noticed the logger was passed from BuilderBase to BaseExecutor. There were few issues related to logging in #265 which i wanted to address later.

This PR was meant to address #256

Reply:
I don't think it makes sense to have a log stream on dry run. We are doign it for time being https://github.com/shahzebsiddiqui/buildtest-framework/blob/66e49c9edaba674d06442ebfed94d9fd549c41af/buildtest/buildsystem/base.py#L528

Reply:
Was this added back, so dry run has stdout logging? Just want to verify before resolving.